noetic-3.17/ros-core/.template.Dockerfile

```diff
@@ -5,5 +5,6 @@ RUN apk add --no-cache py3-rosdep \
   && sed -i -e 's/ros\/rosdistro\/master/alpine-ros\/rosdistro\/alpine-custom-apk/' /etc/ros/rosdep/sources.list.d/20-default.list
 
 RUN apk add --no-cache \
+  py3-rosdistro\>=0.9.0 \
```

Comment:
This should be in the previous `RUN`, otherwise previous layer with old py3-rosdistro will be reused and updated in this layer.